rust-connectors/sinks/kafka/src/main.rs

```diff
@@ -10,6 +10,7 @@ use tokio_stream::StreamExt;
 
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
+    init_open_telemetry();
```

Comment:
Move this under like `common_initialize` so we can consolidate into a single initialization.  In fact, we should have common connector `main` like `fluvio future` (let's take as another issue for that).

Reply:
done